Assignment4/quadrilateral.go

```diff
@@ -0,0 +1,101 @@
+/*
+	3. The given program takes an integer value as input from the user i.e 1 or 2.
+	Option 1 represents Rectangle and Option 2 represents Square.
+
+	Given the metrics of the shapes are hard-coded, complete the given program to achieve the following:
+
+	1. Create an interface Quadrilateral which has the following method signatures
+	* Area() int
+	* Perimeter() int
+
+	2. Implement the Quadrilateral interface for the given shapes i.e Rectangle and Square
+
+	3. Define a "Print" function which accepts any shape that implements Quadrilateral interface and Prints Area and Perimeter of shape in the following manner:
+
+	"Area :  <value>"
+	"Perimeter :  <value>"
+	HINT: Step 2 means, to define methods in Quadrilateral interface for given shapes
+
+	Formulae:
+
+	Area of Rectangle: length * width
+	Perimeter of Rectangle: 2*(length + breadth)
+
+	Area of Square: side * side
+	Perimeter of Square: 4 * side
+
+*/
+
+package main
+
+import "fmt"
+
+type Quadrilateral interface {
+	Area() int
+	Perimeter() int
+}
+
+type Square struct {
```

Comment:
If we aren't exporting any variable to another module, then it should be in small-case camel-casing

Reply:
done